contracts/hardhat.config.ts

```diff
@@ -43,6 +43,7 @@ function createTestnetConfig(network: keyof typeof chainIds): NetworkUserConfig
   return {
     accounts: [(process.env.DEPLOY_PRIVATE_KEY as string) || dummyPrivateKey],
     chainId: chainIds[network],
+    allowUnlimitedContractSize: true,
```

Comment:
@thelostone-mc @apbendi @phutchins why did we need to add this flag?

Reply:
this was because we thought that the contract size was too large - the issue was with startTime (this is deleted in code cleanup commit)